tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -110,6 +112,7 @@ public interface IHaveInterestingProperties
                 .x(() => exception.Message.Should().Be("The specified expression is not a method call or property getter."));
         }
 
+        [SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object)", Justification = "Not relevant in this case")]

```

Comment:
In this case, using `FormattableString.Invariant` is also possible, but would be a little awkward, because the interpolated string is a step description, so it would read `Invariant($"When I assign the property to {value})"` (assuming `FormattableString` is imported with `using static`).